src/main/java/org/sagebionetworks/web/client/widget/entity/download/Uploader.java

```diff
@@ -1025,24 +1027,27 @@ private void uploadSuccess() {
     view.resetToInitialState();
     resetUploadProgress();
     if (successHandler != null) {
-      synapseClient.getEntityBenefactorAcl(
-        parentEntityId,
```

Comment:
For some reason, I tried to use `parentEntityId`, which is null sometimes (maybe I assumed `entityId` might be null?). We can just use the current entity ID.